benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -21,7 +21,7 @@ import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
 
 //example data for development
 if (process.env.NODE_ENV !== "production") {
-  window.data = require("../data/data.json");
+  window.data = require("@data");
```

Comment:
The alias is certainly not a bad idea, but I wonder whether it is overkill here. To understand what file is loaded, one has to first look at the alias definition, and then at the environment variable, so a double indirection.

Can we use either the alias or the environment variable instead of both?

Reply:
I went with a dynamic bind of `@data` in the webpack config, as we are unable to require modules from a env variable